src/pages/Login.tsx

```diff
@@ -0,0 +1,225 @@
+                            alignItems: 'center',
+                        }}
+                    >
+
+                        <Box component="div" sx={{ width: "228px", height: "84px" }}>
+                            <Box
+                                component="img"
+                                src={intercars_logo}
+                                alt="intercars-logo"
+                                sx={{
+                                    display: 'block',
+                                    width: '100%',
+                                    height: 'auto',
+                                    marginBottom: '1.5em'
+                                }}
+                            />
+                        </Box>
+
+                        <Box component="div" sx={{
+                            fontSize: '1.5em',
+                            fontWeight: 800,
+                            padding: '0 2em',
+                            textAlign: 'center',
+                            margin: '2em 0',
+                        }}>
+                            Вътрешна система за менажиране на доставки
+                        </Box>
+
+
+                        <Typography component="h1" variant="h5" sx={{
+                            fontSize: '2em',
+                            fontWeight: 800,
+
+                        }} >
+                            Вход
+                        </Typography>
+                        <Box component="form" onSubmit={handleSubmit(onSubmit)} sx={{ mt: 1 }}>
+                            <Controller
+                                name="email"
+                                control={control}
+                                defaultValue=""
+                                render={({ field }) => (
+                                    <TextField
+                                        {...field}
+                                        label="Имейл"
+                                        id="email"
+                                        name="email"
+                                        variant="outlined"
+                                        margin="normal"
+                                        required
+                                        fullWidth
+                                        autoComplete="email"
+                                        autoFocus
+                                        color="secondary"
+                                        error={!!errors.email}
+                                        helperText={errors.email ? errors.email.message : ''}
+                                    />
+                                )}
+                            />
+
+                            <Controller
+                                name="password"
+                                control={control}
+                                defaultValue=""
+                                render={({ field }) => (
+                                    <TextField
```

Comment:
lets create this hide/show functionality into its own component.

So you can extract this with the logic on switching between the `type`, but have props like `label`, `value` etc.

Reply:
fixed